metrics/metrics.go

```diff
@@ -2,6 +2,7 @@ package metrics
 
 import (
 	"fmt"
+	"github.com/zalando/skipper/filters"
```

Comment:
please keep coupling low. It is not worth importing the filters package for this. In Go having that limited number of interface methods multiple times: is fine.

Reply:
If you say so. Restoring (and adding the method that I did add to `skipper/filters`).

Reply:
solved in 723c857910239b97842c4cf9cb74f26d3f6fe21e